File: solutions/julia/darts/1/darts.jl
function score(x, y)
    d = dist_to_center(x, y)
    if d <= 1
        return 10
    elseif d <= 5
        return 5
    elseif d <= 10
        return 1
    else
        return 0
    end
end

function dist_to_center(x, y)
    sqrt(x ^ 2 + y ^ 2)
end

File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double d = distance(xOfDart, yOfDart);
        if (d > 10) {
            return 0;
        } else if (d > 5) {
            return 1;
        } else if (d > 1) {
            return 5;
        } else {
            return 10;
        }
    }

    private double distance(double x, double y) {
        return Math.sqrt(Math.pow(x, 2.0) + Math.pow(y, 2.0));
    }
}
